### ISSUE TITLE
Vue 3 using composition API throws error cannot read property 'apply' of undefined

### ISSUE THREAD

**Issue by jaylinsell** [stalniy/casl#530]
**Describe the bug**
Using Vue 3's composition API, the steps provided in the documentation throws an error "Cannot read property of undefined".
It works using Vue's inject method with Options API.

**To Reproduce**
1. Install Vue 3, and add casl's dependancies
2. Create ability.js file with conditions, and import/add relevant data to main.js
3. In component, use composition API and use the following code sample as provided in https://casl.js.org/v5/en/package/casl-vue#the-plugin:
```html
<template>
   <div v-if="can('read', 'Post')">Test</div>
</template>
```

```js
import { useAbility } from '@casl/vue';

export default {
  setup() {
    // some code
    const { can } = useAbility();

    return {
      // other props
      can
    };
  }
```

**Expected behavior**
For the div wrapper to only show if the ability condition is met.

**Interactive example** (optional, but highly desirable)
https://codesandbox.io/s/dazzling-mendeleev-43qt5?file=/src/App.vue

**CASL Version**

`@casl/ability` - v5.3.1
`@casl/vue` - v2.1.1

**Environment**:
Node: 14.17.0
Vue 3

**Comment by SergeyMiracle** [comment]
got same error

**Comment by stalniy** [comment]
yes, it looks like `this` is not bound in `can` function, the workaround is to use this:

```js
setup() {
  const ability = useAbility()

  return { can: ability.can.bind(ability) }
}
```

PRs are very welcome, should be easy change
